libraries/engage/locations/components/FulfillmentSelector/FulfillmentSelectorItemDirectShip.connector.js

```diff
@@ -1,13 +0,0 @@
-import { connect } from 'react-redux';
-import { hasProductVariants, isProductOrderable } from '@shopgate/engage/product';
-
-/**
- * @param {Object} state .
- * @param {Object} props .
- * @return {Object}
- */
-const mapStateToProps = (state, props) => ({
-  isOrderable: isProductOrderable(state, props) || hasProductVariants(state, props),
```

Comment:
This state is needed to show error message `locations.fulfillment.error.direct_ship`

Reply:
Done